test/functional/tests/cli/test_zero_metadata_command.py

```diff
@@ -0,0 +1,258 @@
+            cache_disk, core_disk = prepare_devices()
+
+    with TestRun.step("Start cache."):
+        cache = casadm.start_cache(cache_disk, CacheMode.WB, force=True)
+        core = cache.add_core(core_disk)
+        cache.set_cleaning_policy(CleaningPolicy.nop)
+
+    with TestRun.step("Run workload on CAS"):
+        fio_run_fill = Fio().create_command()
+        fio_run_fill.io_engine(IoEngine.libaio)
+        fio_run_fill.direct()
+        fio_run_fill.read_write(ReadWrite.randwrite)
+        fio_run_fill.io_depth(16)
+        fio_run_fill.block_size(Size(1, Unit.MebiByte))
+        fio_run_fill.target(core.path)
+        fio_run_fill.run_time(timedelta(seconds=5))
+        fio_run_fill.time_based()
+        fio_run_fill.run()
+
+    with TestRun.step("Stop cache without flushing dirty data."):
+        cache.stop(no_data_flush=True)
+
+    with TestRun.step("Start cache (expect to fail)."):
+        try:
+            cache = casadm.start_cache(cache_disk, CacheMode.WB)
+        except CmdException:
+            TestRun.LOGGER.info("Start cache failed as expected.")
+
+    with TestRun.step("Zero metadata on CAS device without force"):
+        output = TestRun.executor.run(cli.zero_metadata_cmd(str(cache_disk.path)))
+        if output.exit_code != 0:
+            cli_messages.check_stderr_msg(output, cli_messages.cache_dirty_data)
+        else:
+            TestRun.LOGGER.error("Zeroing metadata without force should fail!")
+
+    with TestRun.step("Zero metadata on cache device with force"):
+        output = TestRun.executor.run(cli.zero_metadata_cmd(str(cache_disk.path), force=True))
+        if output.exit_code == 0:
+            TestRun.LOGGER.info("Zeroing metadata with force ran successfully!")
+        else:
+            TestRun.LOGGER.error("Zero metadata with force should works for cache device!")
+
+        with TestRun.step("Start cache."):
+            try:
+                cache = casadm.start_cache(cache_disk, CacheMode.WB)
+                TestRun.LOGGER.info("Cache started successfully.")
+            except CmdException:
+                TestRun.LOGGER.error("Start cache failed.")
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_zero_metadata_dirty_shutdown():
+    """
+        title: Test for '--zero-metadata' and dirty shutdown scenario.
+        description: |
+          Test for '--zero-metadata' with and without 'force' option on cache which had been dirty
+          shut down before.
+        pass_criteria:
+          - Zero metadata without force failed on cache after dirty shutdown.
+          - Zero metadata with force ran successfully on cache after dirty shutdown.
+          - Cache started successfully after operations.
+    """
+    with TestRun.step("Prepare cache and core devices."):
+        with TestRun.step("Prepare devices."):
+            cache_disk = TestRun.disks['cache']
```

Comment:
You could use `prepare_devices()` function here and just make it return the cache_disk for unplugging, you could just ignore this in other tests